client/state/data-layer/wpcom/comments/index.js

```diff
@@ -36,6 +63,38 @@ export const fetchPostComments = ( { dispatch, getState }, action ) => {
 	}, action ) );
 };
 
+export const writePostComment = ( { dispatch }, action ) => {
+	const { siteId, postId, parentCommentId, commentText } = action;
+	const placeholder = createPlaceholderComment( commentText, postId, parentCommentId );
+
+	// Insert a placeholder
+	dispatch( {
+		type: COMMENTS_RECEIVE,
+		siteId,
+		postId,
+		comments: [ placeholder ],
+		skipSort: !! parentCommentId
+	} );
+
+	const path = !! parentCommentId
+		? `/sites/${ siteId }/comments/${ parentCommentId }/replies/new`
+		: `/sites/${ siteId }/posts/${ postId }/replies/new`;
```

Comment:
on the naming, I'm not entirely sure where it ought to go, but I'm pretty sure not here

could we start with this new trend and use the `resources` suggestion above? do you have hesitations against that?